src/components/ClassCard/ClassCard.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import "./ClassCard.css";
+import { IoRemoveCircle } from "react-icons/io5";
+
+export default function ClassCard({
+  id,
+  name,
+  duration,
+  intervalTime,
+  onRemoveClassClick,
+}) {
+  return (
+    <section className="ClassCard">
+      <h2>{name}</h2>
+      <div className="ClassCard__input">
+        <p>Duration</p>
+        <p>{duration} min</p>
+        <div className="ClassCard--removeItem">
+          <IoRemoveCircle
+            className="ClassCard-Icon"
+            onClick={() => {
+              onRemoveClassClick(id);
+            }}
+          />
+        </div>
+        <p>Intervals</p>
+        <p>{intervalTime} min</p>
```

Comment:
Since you wrote this in line 16 + 17 as well maybe this could be a sweet component ;-)

Reply:
Haha I will pin this suggestion and write a new issue to my backlog.